gradle.properties

```diff
@@ -20,4 +20,8 @@ kotlin.code.style=official
 # Enables namespacing of each library's R class so that its R class includes only the
 # resources declared in the library itself and none from the library's dependencies,
 # thereby reducing the size of the R class for that library
-android.nonTransitiveRClass=true
\ No newline at end of file
+android.nonTransitiveRClass=true
+
+# Let test include roborazzi verification
+# https://github.com/takahirom/roborazzi?tab=readme-ov-file#roborazzitest
+roborazzi.test.verify=true
```

Comment:
Oh, nice one. Should we remove the explicit call for this on the GH action?

Reply:
Since there are no other test, I think we can keep the action like it is.

Reply:
But won't it run the tests twice (unit + screenthots, then screenshots)?

Reply:
I mean, we have:

```
- name: ⚙️ Run unit tests
  run: ./gradlew testDebugUnitTest $CI_GRADLE_ARG_PROPERTIES

- name: 📸 Run screenshot tests
  run: ./gradlew :compound:verifyRoborazziDebug $CI_GRADLE_ARG_PROPERTIES
  ```
  
  If we're including the screenshot tests in the unit ones we don't need to have both steps.